README.md

```diff
@@ -6,4 +6,39 @@
 
 ### Development
 
-* Styling: Tailwind and Flowbite
\ No newline at end of file
+* Styling: Tailwind and Flowbite
+
+### Calling and Using Models
+
+Models can be called directly from the fileserver with the following url:
+
+```
+Headers: X-DreamFactory-Api-Key: {YOUR_FILESERVER_APP_API_KEY}
+GET: {FILESERVER_URL}/files/models/{MODEL_NAME}/?zip=true
+```
+An example curl request:
+
+```
+curl -X GET "http://18.183.51.71/api/v2/files/models/TintinRocket/?zip=true"  -H "X-DreamFactory-API-Key: {API_KEY_HERE}" -o myfilename.zip
+```
+
+Note that the downloaded zip will include the parent folder "models" so you will need to move over the model_name directory to your simulator assets path.
+
+#### Fuel Models
+
+Models added from Fuel will extract in the regular "fuel" manner and so can be called with `model.sdf` e.g in the case of Gazebo
+```
+<model name="my_model">
+    <include>
+        <uri>models://my_model_name/model.sdf</uri>
```

Comment:
actually, when including a model, you only need to refer to the model folder, and it is not needed to point to the sdf file

Reply:
Thank you, changed